test/make.sh

```diff
@@ -62,81 +58,110 @@ function basefiles() {
   done
 }
 
-# This function runs the hadolint linter on
-# every file named 'Dockerfile'
-function docker() {
-  echo "Running hadolint on Dockerfiles"
-  find_files . -name "Dockerfile" -print0 \
-    | compat_xargs -0 hadolint
+function check_bash() {
+find . -name "*.sh" | while IFS= read -d '' -r file;
+do
+  if [[ "$file" != *"bash -e"* ]];
+  then
+    echo "$file is missing shebang with -e";
+    exit 1;
+  fi;
+done;
+}
+
+# This function makes sure that the required files for
+# releasing to OSS are present
+function basefiles() {
+  echo "Checking for required files"
+  test -f LICENSE || echo "Missing LICENSE"
+  test -f README.md || echo "Missing README.md"
 }
 
 # This function runs 'terraform validate' against all
-# directory paths which contain *.tf files.
+# files ending in '.tf'
+
 function check_terraform() {
```

Comment:
why is't different from https://github.com/terraform-google-modules/terraform-google-project-factory/blob/32a539ab439ca8610db073091594f32a5768c21c/test/make.sh#L84 ?

Reply:
Because this solution is optimal

Reply:
My suggestion doesn't try to test terraform.tfvars and it's symlinks by `terraform fmt`